sdk/webpubsub/azure-messaging-webpubsub/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
+## 1.0.0 (2021-11-23)
 
-### Features Added
-
-### Breaking Changes
+### Other Changes
```

Comment:
In the "Breaking Changes" section, we should call out that `setErrorOptions` is removed from `RequestOptions` even though this change does not belong directly in this package, it is worth mentioning it here as RequestOptions is the primary type users interact with.